cluster/engine.go

```diff
@@ -18,6 +18,7 @@ import (
 	log "github.com/Sirupsen/logrus"
 	"github.com/docker/docker/pkg/version"
 	engineapi "github.com/docker/engine-api/client"
+	apitypes "github.com/docker/engine-api/types"
```

Comment:
nit: last one, sorry missed this earlier `apitypes` -> `types` :)

Everything else is good!